src/test/java/uk/gov/companieshouse/service/CompanyOfficerValidatorTest.java

```diff
@@ -170,4 +207,76 @@ public void areMajorityOfCompanyOfficersSelected_lessThanHalfDirectorsSelected_r
 
         assertFalse(hasOverHalfDirectorsSelected);
     }
+
+    @Test
+    public void areMajorityOfLLPCompanyOfficersSelected_halfLLPMembersSelected_returnsTrue() {
```

Comment:
half would and should fail. More than half would pass. Looks like this test is checking 2 out of 3 are selected.